pkg/utils/configs/config.go

```diff
@@ -115,7 +131,11 @@ func SetInterconnectDefaults(m *v1alpha1.Interconnect, certMgrPresent bool) (boo
 			if certMgrPresent {
 				m.Spec.InterRouterListeners = append(m.Spec.InterRouterListeners, v1alpha1.Listener{
 					Port:       55671,
-					SslProfile: "default",
+					SslProfile: "inter-router-tls",
```

Comment:
Had one minor comment, is the `-tls` needed in the name? Generated cred ends up with `-tls-tls`

Reply:
agreed, I've changed that